apps/mobile/src/screens/ResultsScreen.js

```diff
@@ -1,21 +1,16 @@
 // @flow
 
 import * as React from 'react';
-import {
-  AnywhereResults,
-  EmptyResults,
-  ResultsList,
-} from '@kiwicom/margarita-core';
+import { Results } from '@kiwicom/margarita-core';
 
 type Props = Object;
```

Comment:
I know that it is not part of your PR but is possible write this types properly?

Reply:
Ok, types updated. So it's currently more clear which parameters component expects.
Later with navigation update this type will be probably updated.

Reply:
thanks